packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -342,8 +342,6 @@ const ruleForBabelLoader = (babelPlugins) => {
         id: 'babel-loader',
         test: /(\.js(x?)|\.ts(x?))$/,
         // NOTE: Because our extensions are just folders containing source code, we need to ensure that the babel-loader processes them.
-        // By default babel doesn't process files in "node_modules" folder, so here we will ensure they are included.
```

Comment:
Can you check that when you build and deploy on mrt that the extend server is still happening. Just wanted to do a gut check here.

Reply:
Yes, still works. I deployed a bundle with the `extension-sample`.

https://scaffold-pwa-test-env-3.mobify-storefront.com/sample